db/schema.rb

```diff
@@ -174,9 +173,9 @@
     t.datetime "deleted_at"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.uuid "region_id"
+    t.uuid "region_id", null: false
+    t.uuid "facility_id"
```

Comment:
Do we keep this around to preserve older data? Can we get away with _not_ having to do that? A data migration plus dropping the column?

Reply:
We want to do this eventually but it requires more refactoring across the view and the controller. We can do this iteratively, don't want to block releasing this on the cleanup.

Reply:
Can you craft a story and link it here?